File: Data Structures & Algorithms/duplicate-integer/submission-3.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {boolean}
     */
    hasDuplicate(nums) {
        const store = {}
        for (const num of nums){
            if(store[num]) {
                return true
            }
            store[num] = 1
        }
        return false
    }
}

File: Data Structures & Algorithms/is-anagram/submission-3.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s) != len(t):
            return False
        return sorted(s) == sorted(t)
        # store = []
        # for str in s:
        #     store.append(str)
        # for str in t:
        #     if str in store:
        #         store.remove(str)
        #     else:
        #         store.append(str)

        # return len(store) == 0        